firstvoices/backend/search/utils/site_signals.py

```diff
@@ -180,3 +203,8 @@ def remove_song_from_index(song):
                 song.id,
             )
         )
+    except Exception as e:
+        # Fallback exception case
+        logger = logging.getLogger(ELASTICSEARCH_LOGGER)
+        logger.error(type(e).__name__, SearchIndexEntryTypes.SONG, song.id)
+        logger.error(e)
```

Comment:
Do we need matching signals in here for stories? No need to add right now, but please add a ticket if necessary